protoc-gen-grpc-gateway/descriptor/registry.go

```diff
@@ -318,6 +325,22 @@ func (r *Registry) SetMergeFileName(mergeFileName string) {
 	r.mergeFileName = mergeFileName
 }
 
+// SetAllowRepeatedFieldsInBody controls whether repeated field can be used in body field path
+// of `google.api.http` annotation option or not
+func (r *Registry) SetAllowRepeatedFieldsInBody(allow bool) {
+	r.allowRepeatedFieldsInBody = allow
+}
+
+// SetUseJSONNameInSwaggerDef sets useJSONNameInSwaggerDef
+func (r *Registry) SetUseJSONNameInSwaggerDef(use bool) {
+	r.useJSONNameInSwaggerDef = use
+}
+
+// GetUseJSONNameInSwaggerDef whether generation one swagger file out of multiple protos
```

Comment:
This comment is incorrect and nonsensical*, could you please fix it? :)

* Its grammar is nonsensical even in `IsAllowMerge`, but here the comment itself is outright wrong.